src/current/v23.2/set-up-physical-cluster-replication.md

```diff
@@ -0,0 +1,313 @@
+
+The primary cluster requires the following value:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+--config-profile replication-source
+~~~
+
+For example, a `cockroach start` command according to the [prerequisite deployment guide]({% link {{ page.version.version }}/deploy-cockroachdb-on-premises.md %}#step-3-start-nodes):
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach start \
+--certs-dir=certs \
+--advertise-addr=<node1 address> \
+--join=<node1 address>,<node2 address>,<node3 address> \
+--cache=.25 \
+--max-sql-memory=.25 \
+--background \
+--config-profile replication-source
+~~~
+
+### Connect to the primary cluster system interface
+
+Connect to your primary cluster's system interface using [`cockroach sql`]({% link {{ page.version.version }}/cockroach-sql.md %}).
+
+1. To connect to the system interface, pass the `options=-ccluster=system` parameter in the URL:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cockroach sql --url \
+    "postgresql://root@{your IP or hostname}:26257/?options=-ccluster=system&sslmode=verify-full" \
+    --certs-dir "certs"
+    ~~~
+
+    You will find that the prompt will include `system` when you have successfully connected.
+
+    {{site.data.alerts.callout_info}}
+    You should only use the system interface for administration. To work with databases, tables, or workloads, use the application virtual cluster{% comment %}({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}){% endcomment %}.
+    {{site.data.alerts.end}}
+
+1. Add your cluster organization and {{ site.data.products.enterprise }} license to the cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING cluster.organization = 'your organization';
+    ~~~
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING enterprise.license = 'your enterprise license';
+    ~~~
+
+1. Confirm the status of your virtual cluster: {% comment %}Link to SQL ref page here{% endcomment %}
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTERS;
+    ~~~
+
+    The output will include the `system` interface and the `application` virtual cluster:
+
+    ~~~
+    id |    name     | data_state | service_mode
+    ---+-------------+------------+---------------
+    1  | system      | ready      | shared
+    2  | template    | ready      | none
+    3  | application | ready      | shared
+    (3 rows)
+    ~~~
+
+### Create a replication user and password
+
+The standby cluster requires a user profile on the primary cluster to connect to.
+
```

Comment:
I think at the same time you should have them create a DB Console user on the primary cluster. That way, when the cut over to the standby, they can immediately use that identity to log in and verify that all the databases they expect to see in the application cluster, are present.

Reply:
I think this was captured in one of the steps where I have now created a users.